packages/ui/src/components/organisms/Footer/styles.scss

```diff
@@ -20,6 +20,9 @@
   --fs-footer-title-weight                        : var(--fs-text-weight-bold);
   --fs-footer-title-margin-bottom                 : var(--fs-spacing-1);
 
+  // Logo
+  --fs-footer-logo-width                          : var(--fs-logo-width);
```

Comment:
should we add it to the docs?

Reply:
We don't have doc for the `Footer` component

Reply:
Because we haven't moved it to `fs/components` 😬

Reply:
ohh, that's true!